doc/developers/contributing.rst

```diff
@@ -389,6 +390,28 @@ and Cython optimizations.
      [doc build]            Docs built including example gallery plots
      ====================== ===================
 
+Stalled pull requests
+......................
+
+As contributing a new feature can be a lengthy process, some
+pull requests appear inactive but unfinished. In such a case, taking
+them over is a great service for the project.
+
+A good etiquette for this is:
+
```

Comment:
A pull request may have the label "stalled" or "help wanted" if we have already identified it as a candidate for other contributors.

Reply:
I add a note about these.